### PR TITLE
different behaviour while updating using grpc.gateway.protoc_gen_swagger.options.openapiv2_operation 

### PR DESCRIPTION
**Pull request by msannareddybloxer**
Issue:
  while updating the responses using grpc.gateway.protoc_gen_swagger.options.openapiv2_operation is not working properly.

### CODE REVIEW

File: protoc-gen-swagger/genswagger/atlas_patch.go [hunk 1]
```diff
@@ -230,6 +230,7 @@ The service-defined string used to identify a page of resources. A null value in
 							op.Responses.StatusCodeResponses[opToStatusCode(on)] = rsp
 							delete(sw.Definitions, trim(rsp.Ref))
 							delete(op.Responses.StatusCodeResponses, 200)
+							delete(op.Responses.StatusCodeResponses, 201)
```
Comment (burov) [inline protoc-gen-swagger/genswagger/atlas_patch.go:233]: Solution fix current problem, but can bring new some time in a the future f.e. someone return 202 or somethis like that. Is it better to make a general solution, just some function which clean up a whole range of success statuses 2** or something like that

**Comment by burov** [comment]
In case of urgency i think we can go with this solution, but have to keep in mind that problem/

File: protoc-gen-swagger/genswagger/atlas_patch.go (hunk 1 above)
Comment (msannareddybloxer) [reply protoc-gen-swagger/genswagger/atlas_patch.go:233]: @burov i have seen this comment as part of atlas-patch "Ability to wrap Responses with correct error codes (200 - for GET, 201 - for POST/PUT/PATCH, 204 - for DELETE)".  So i have removed 201 alone.

**Comment by msannareddybloxer** [comment]
> 
> 
> In case of urgency i think we can go with this solution, but have to keep in mind that problem/

I agree, can we go with this for now as we are blocked with hostapp release

**Review by burov — APPROVED** [review]
Yes sure, we can go with this solution, thank you for contribution

**Pull request merged**